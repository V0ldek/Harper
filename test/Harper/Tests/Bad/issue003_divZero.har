main :: Integer;
main = 42 / 0;