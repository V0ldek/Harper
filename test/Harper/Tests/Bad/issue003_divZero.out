Error: expression `0` evaluated to zero causing a division error.
During evaluation of:
 42 / 0
Located at line 2 column 8

Execution terminated with an error: runtime error.